lib/metadata_gen.js

```diff
@@ -31,15 +32,24 @@ class MetadataGenerator {
     const parser = new LinkParser(owner, repo, op);
     const fixes = parser.getFixes();
     const refs = parser.getRefs();
-
+    const altUrl = parser.getAltPrUrl();
```

Comment:
I find it confusing when the same thing has slightly different names, `altUrl` is not obviously the result of `getAltPrUrl()`, but it would be if it was named `altPrUrl`.